.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+Resolves <!-- paste issue reference -->
+<!-- and/or -->
+Part of <!-- paste issue/PR reference -->
+<!-- and/or -->
+Required for <!-- paste issues/PRs references -->
+<!-- and/or -->
+Requires <!-- paste issues/PRs references -->
+<!-- and/or -->
+Related to <!-- paste issues/PRs references -->
+
+<!-- Remove the lines above if there are no related issues/PRs -->
+
+
+
+
+## Synopsis
+
+<!-- Give a brief overview of the problem -->
+
+
+
+
+## Solution
+
+<!-- Describe how exactly the problem is (or will be) resolved -->
+
+
+
+
+## Checklist
+
+- [ ] Documentation is updated (if required)
+- [ ] Tests are added/updated (if required)
+- [ ] [CHANGELOG entry][l:1] is added (if required)
+- [ ] [Review][l:2] is completed and changes are approved
```

Comment:
I don't think this is necessary in the list. The branch protection rules already ensure this. This checklist seems mostly useful for contributors/reviewers not to forget things.

Reply:
Done in https://github.com/JelteF/derive_more/pull/212/commits/ca7696772337598d937e870b6403d45b4945faf9.